desktop/cmp/button/RefreshButton.js

```diff
@@ -9,41 +9,66 @@ import {Component} from 'react';
 import {PropTypes as PT} from 'prop-types';
 import {elemFactory, HoistComponent} from '@xh/hoist/core';
 import {Icon} from '@xh/hoist/icon';
-import {button} from '@xh/hoist/kit/blueprint';
+import {button} from './Button';
+import {warnIf} from '@xh/hoist/utils/JsUtils';
 
 /**
  * Convenience Button preconfigured for use as a trigger for a refresh operation.
  * Accepts props documented below as well as any others supported by Blueprint's Button.
  *
- * Must be provided either an onClick handler *or* a model. If a model is provided, this button
- * will call loadAsync() on the model class.
+ * Must be provided either an onClick handler *or* a model. If a model is provided and an onClick
+ * handler is not provided, this button will call loadAsync() on the model class.
  */
 @HoistComponent()
 export class RefreshButton extends Component {
 
     static propTypes = {
+        /** Icon to display for the button. Defaults to Icon.sync(). */
         icon: PT.element,
+        /** Tooltip text to display when the mouse is over the button. Defaults to 'Refresh'. */
         title: PT.string,
+        /** Function to call when the button is clicked. */
         onClick: PT.func,
+        /**
+         * Model to call loadAsync() on when the button is clicked. Only used if onClick was not
+         * provided.
```

Comment:
Sorry to nitpick you here 😄because I really do appreciate you keeping a consistent line length throughout this change. I usually try to avoid the one-word-on-last-line wrap if I can. A good approach can be to shorten up the comment to avoid the wrap if you can still keep it sensible, e.g. "Model to refresh via loadAsync(), if onClick prop not provided."